newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -247,9 +250,12 @@ export const onRenderNewProjectSaveAsLocationChooser = ({
     : newProjectsDefaultFolder
     ? newProjectsDefaultFolder
     : '';
+  const projectFileName = projectName
+    ? cleanUpIdentifier(projectName, 1, 60) + '.json'
+    : 'game.json';
```

Comment:
Should non-latin characters be allowed?
It may work and be more user-friendly but it can probably cause issues too.

Reply:
I think we can only prevent "non-allowed" characters, that can cause issues on some OS (windows in particular)
We've done something similar when generating the game:

https://github.com/4ian/GDevelop/blob/81bce617836326fd8d0346de8efdb499f4dbd042/GDJS/GDJS/IDE/ExporterHelper.cpp#L72-L93